it/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -56,94 +56,94 @@
 
 ## @name NS_MSG_NO_RECIPIENTS
-12511=Non sono stati specificati destinatari. Inserire un destinatario o un gruppo di discussione nella sezione degli indirizzi.
+12511=Non sono stati specificati destinatari. Inserisci un destinatario o un newsgroup nell'area indirizzo.
 
 ## @name NS_MSG_ERROR_WRITING_FILE
 12512=Errore di scrittura sul file temporaneo.
 
 ## @name NS_ERROR_SENDING_FROM_COMMAND
-12514=Si è verificato un errore durante l'invio della posta. Il server di posta ha risposto: %s. Verificare che il proprio indirizzo e-mail nelle impostazioni di posta sia corretto e riprovare.
+12514=Si è verificato un errore durante l'invio della e-mail. Il server di posta ha risposto: %s. Verifica nelle impostazioni e-mail che il tuo indirizzo e-mail sia corretto e riprova.
 
 ## @name NS_ERROR_SENDING_DATA_COMMAND
-12516=Si è verificato un errore (SMTP) durante l'invio della posta. Il server ha risposto: %s.
+12516=Si è verificato un errore (SMTP) durante l'invio della e-mail. Il server ha risposto: %s.
 
 ## @name NS_ERROR_SENDING_MESSAGE
-12517=Si è verificato un errore durante l'invio della posta. Il server di posta ha risposto: %s. Controllare il messaggio e riprovare.
+12517=Si è verificato un errore durante l'invio della e-mail. Il server di posta ha risposto: %s. Controlla il messaggio e riprova.
 
 ## @name NS_ERROR_POST_FAILED
-12518=Il messaggio non può essere pubblicato a causa della mancata connessione al server delle news. Il server potrebbe non essere raggiungibile o rifiutare la connessione. Ricontrollare le impostazioni del server delle news o contattare il proprio amministratore di sistema.
+12518=Il messaggio non può essere pubblicato poiché la connessione al server delle news non è riuscita. Il server potrebbe non essere disponibile o rifiutare la connessione. Verifica che le tue impostazioni del server news siano corrette oppure contatta il tuo Amministratore di sistema.
 
 ## @name NS_ERROR_QUEUED_DELIVERY_FAILED
 12519=Si è verificato un errore nella consegna dei messaggi da inviare.
 
 ## @name NS_ERROR_SEND_FAILED
-12520=Non è riuscito l'invio del messaggio.
+12520=Invio del messaggio non riuscito.
 
 ## @name SMTP_DELIV_MAIL
-12521=Consegna posta in corso…
+12521=Consegna e-mail in corso…
 
 ## @name SMTP_MESSAGE_SENT_WAITING_MAIL_REPLY
-12522=Posta: Messaggio inviato; in attesa di risposta…
+12522=E-mail: Messaggio inviato; in attesa di risposta…
 
 ## @name SMTP_PROGRESS_MAILSENT
-12523=Posta inviata con successo
+12523=E-mail inviata con successo
 
 ## @name NS_ERROR_SMTP_SERVER_ERROR
-12524=Si è verificato un errore durante l'invio della posta: Errore del server SMTP. Il server ha risposto: %s Contattare l'amministratore del server di posta per assistenza.
+12524=Si è verificato un errore durante l'invio della e-mail: errore del server SMTP. Il server ha risposto: %s Contatta il tuo Amministratore e-mail per assistenza.
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

 +12524=Si è verificato un errore durante l'invio dell'e-mail: errore del server SMTP. Il server ha risposto: %s Contatta l'Amministratore delle e-mail per assistenza.